doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-run-test-proc.adoc

```diff
@@ -1,13 +1,22 @@
 [id='test-designer-run-test-proc']
-= Running a test scenario
+= Running the test scenarios
```

Comment:
Question - when pluralizing titles, do we usually include the article?

Reply:
Yes, we include the article in the title.